src/coreclr/System.Private.CoreLib/CompatibilitySuppressions.xml

```diff
@@ -29,4 +29,10 @@
     <DiagnosticId>CP0015</DiagnosticId>
     <Target>M:System.Diagnostics.Tracing.EventSource.WriteEventWithRelatedActivityIdCore(System.Int32,System.Guid*,System.Int32,System.Diagnostics.Tracing.EventSource.EventData*):[T:System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute]</Target>
   </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0016</DiagnosticId>
+    <Target>M:System.Reflection.Assembly.LoadFrom(System.String,System.Byte[],System.Configuration.Assemblies.AssemblyHashAlgorithm):[T:System.ObsoleteAttribute]</Target>
```

Comment:
Can these CompatibilitySuppressions be listed just once in https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/CompatibilitySuppressions.xml since they are identical for all CoreLib variants?

Reply:
Aren't these suppressions only needed because there's a difference between ref and def assembly? I would think after updating the ref assembly, these won't be needed anymore. I'm a bit suspicious about the other suppressions in this file too.

Reply:
IIRC, these suppressions are sometimes needed to make the build happy about vPrevious -> vNext deltas. I am not sure whether it is the case here. I agree that the existing suppressions in this file look suspicious and they should not be needed.

Reply:
I can certainly try removing the suppression now that I've updated the ref assembly.